### PR TITLE
Create intro, player info splash screens, temp game board, fade trans…

### PR DESCRIPTION
**Pull request by Triplemanus**
…itions

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -18,15 +18,42 @@ import Player from './Player';
 import Round from './Round';
 import Turn from './Turn';
 
+function createPlayers(Player1, Player2, Player3) {
+  let player1 = new Player(Player1, 1)
```
Comment (CLLane) [inline src/index.js]: Why are you passing a second argument on in the new instance, I believe the parameter is score for that argument which defaults to 0.

File: src/index.js [hunk 2]
```diff
@@ -18,15 +18,42 @@ import Player from './Player';
...
+  let player2 = new Player(Player2, 2)
```
Comment (CLLane) [inline src/index.js]: Why are you passing a second argument on in the new instance, I believe the parameter is score for that argument which defaults to 0.

File: src/index.js [hunk 3]
```diff
@@ -18,15 +18,42 @@ import Player from './Player';
...
+  let player3 = new Player(Player3, 3)
```
Comment (CLLane) [inline src/index.js]: Why are you passing a second argument on in the new instance, I believe the parameter is score for that argument which defaults to 0.

File: src/index.js [hunk 4]
```diff
@@ -18,15 +18,42 @@ import Player from './Player';
...
+//   // will first fade out the loading animation
+// $('.welcome-message').delay(1500).fadeOut("slow");
+//   // will fade out the whole DIV that covers the website.
+// //$("#preloader").delay(2000).fadeOut("slow");
+// })
 $(document).ready(function() {
-
-  $('#player1-name__submit').click( () => {
- getPlayerName('#player1-name__input', '#player1-name__display')
+  $('.user-name-inputs').hide();
```
Comment (CLLane) [inline src/index.js]: The DTR is that we are querySelecting with jQuery by #id.

File: src/index.js [hunk 5]
```diff
@@ -18,15 +18,42 @@ import Player from './Player';
...
+  $('.puzzle-table').hide();
```
Comment (CLLane) [inline src/index.js]: The DTR is that we are querySelecting with jQuery by #id.

File: src/index.js [hunk 6]
```diff
@@ -18,15 +18,42 @@ import Player from './Player';
...
+  $('.welcome-message').delay(2500).fadeOut("slow");
```
Comment (CLLane) [inline src/index.js]: The DTR is that we are querySelecting with jQuery by #id.

File: src/index.js [hunk 7]
```diff
@@ -18,15 +18,42 @@ import Player from './Player';
...
+  $('.user-name-inputs').append(`
```
Comment (CLLane) [inline src/index.js]: The DTR is that we are querySelecting with jQuery by #id.

File: src/index.js [hunk 8]
```diff
@@ -18,15 +18,42 @@ import Player from './Player';
...
+  <input type="text"id="player2-name__input"> Player 2 </input>
+  <p></p>
+  <input type="text"id="player3-name__input" label="Player3">Player 3:</input>
+  <p></p>
+  <button id="player3-name__submit">Game on!</button>`).delay(3000).fadeIn('slow');
+  //$('.user-name-inputs').delay(3000).fadeIn("slow");
+
+  $('#player3-name__submit').click( () => {
+ getPlayerName('#player3-name__input', '#player3-name__display');
+ $('.user-name-inputs').fadeOut();
```
Comment (CLLane) [inline src/index.js]: The DTR is that we are querySelecting with jQuery by #id.

File: src/index.js [hunk 9]
```diff
@@ -18,15 +18,42 @@ import Player from './Player';
...
+ $('.puzzle-table').delay(1000).fadeIn();
```
Comment (CLLane) [inline src/index.js]: The DTR is that we are querySelecting with jQuery by #id.

File: src/index.js (hunk 8 above)
Comment (CLLane) [inline src/index.js]: The DTR is that we are querySelecting with jQuery by #id.

File: src/css/base.scss [hunk 10]
```diff
@@ -1,6 +1,41 @@
...
+  height: 300px;
+  width: 600px;
+}
+
+.puzzle-table {
+  margin-left: 33%;
+  height: 60%;
+}
+
+  .tg {
```
Comment (CLLane) [inline src/css/base.scss]: Just for my sake can you tell me what .tg stands for?

Comment (Triplemanus) [reply src/css/base.scss]: Table Generator

File: src/index.js (hunk 8 above)
Comment (Triplemanus) [reply src/index.js]: Identifier changed to ID

File: src/index.js (hunk 9 above)
Comment (Triplemanus) [reply src/index.js]: Identifier changed to ID

File: src/index.js (hunk 8 above)
Comment (Triplemanus) [reply src/index.js]: Identifier changed to ID

File: src/index.js (hunk 7 above)
Comment (Triplemanus) [reply src/index.js]: Identifier changed to ID

File: src/index.js (hunk 6 above)
Comment (Triplemanus) [reply src/index.js]: Changed identifier to ID from class

**Pull request merged**